src/test/operators/join_test_runner.cpp

```diff
@@ -580,6 +660,49 @@ TEST_P(JoinTestRunner, TestJoin) {
     print_configuration_info();
     FAIL();
   }
+
+  if (configuration.index_side) {  // configuration is a specialized JoinIndex configuration
+    std::shared_ptr<const Table> index_table = nullptr;
+    if (configuration.index_side == IndexSide::Left) {
+      index_table = join_op->input_table_left();
+    } else {
+      index_table = join_op->input_table_right();
+    }
+
+    // count how many chunks had to be scanned with index
```

Comment:
Shouldn't it be possible to determine that number from the test parameters?

Reply:
We now provide two different shares of indexed segments: 100% and 10%. cdf931d3370b0c4855cb5173e6616537d6ac18d0

Reply:
I agree. Therefore, `indexed_chunk_count` was added to an `InputTableConfiguration` in a0a199f37952b8a4e0e3680887651d0a31585618